frontend/src/app/components/experiments/experiment-detail/experiment-detail.component.html

```diff
@@ -52,7 +52,7 @@ <h1>
                 {{experimentTemplate.name}}</a>
         </span>
 
-        <div class="experiment-env-vars" *ngIf="experiment.env_vars.length > 0">
+        <div class="experiment-env-vars" *ngIf="experiment.env_vars.length > 0 && experiment.mine">
```

Comment:
I think we can keep the div visible in all the cases since the values are hidden by backend